pvlib/irradiance.py

```diff
@@ -3766,3 +3766,69 @@ def louche(ghi, solar_zenith, datetime_or_doy, max_zenith=90):
         data = pd.DataFrame(data, index=datetime_or_doy)
 
     return data
+
+
+def spitters_relationship(solar_zenith, global_diffuse_fraction):
```

Comment:
I'd prefer to the function name to provide some clue as to its purpose. `diffuse_par_spitters`? Inserting "fraction" doesn't add much and makes it 30% longer.

Alternatively, we could name this `spitters` in a similar way we have `erbs`, `louche`, `dirint` etc.

Reply:
Yeah, I prefer that too. A bit hesitant to use just the author name since that quasi-convention is already used for PV irradiance decomposition.